test/extensions/filters/udp/udp_proxy/udp_proxy_filter_test.cc

```diff
@@ -181,6 +233,37 @@ class UdpProxyFilterTest : public testing::Test {
     EXPECT_EQ(tx_datagrams, config_->stats().downstream_sess_tx_datagrams_.value());
   }
 
+  void checkSocketOptions(TestSession& session, const Network::SocketOptionName& ipv4_option,
+                          int ipv4_expect, const Network::SocketOptionName& ipv6_option,
+                          int ipv6_expect) {
+    EXPECT_EQ(ipv4_expect, session.sock_opts_[ipv4_option.level()][ipv4_option.option()]);
+    EXPECT_EQ(ipv6_expect, session.sock_opts_[ipv6_option.level()][ipv6_option.option()]);
+
+    const auto ipv4_option_details = session.findOptionDetails(
+        ENVOY_SOCKET_IP_TRANSPARENT, envoy::config::core::v3::SocketOption::STATE_PREBIND);
+    const auto ipv6_option_details = session.findOptionDetails(
+        ENVOY_SOCKET_IPV6_TRANSPARENT, envoy::config::core::v3::SocketOption::STATE_PREBIND);
+
+    EXPECT_EQ(ipv4_expect == 1, ipv4_option_details.has_value());
+    EXPECT_EQ(ipv6_expect == 1, ipv6_option_details.has_value());
+  }
+
+  bool isSocketOptionsSupported(const std::vector<Network::SocketOptionName>& option_names) {
```

Comment:
This function is only used in isTransparentSocketOptionsSupported(), inline it please

Reply:
Will be fixed.